src/main/kotlin/me/modmuss50/mpp/ModPublishExtension.kt

```diff
@@ -27,6 +29,13 @@ abstract class ModPublishExtension(val project: Project) : PublishOptions {
     val ALPHA = ReleaseType.ALPHA
     val STABLE = ReleaseType.STABLE
 
+    val CLASSIC = MessageLook.CLASSIC
+    val MODERN = MessageLook.MODERN
+
+    val EMBED = LinkType.EMBED
+    val BUTTON = LinkType.BUTTON
+    val INLINE = LinkType.INLINE
```

Comment:
Why are these needed? I was never happy with how I did this for the release types, id just accept strings in the API.